CoreUtilities/src/au/gov/asd/tac/constellation/utilities/graphics/Mathf.java

```diff
@@ -318,7 +218,9 @@ public static void catmullRom(final float[] vOut, final float[] vP0, final float
      * compare value.
      */
     public static boolean closeEnough(final float candidate, final float compare, final float epsilon) {
-        return Math.abs(candidate - compare) < epsilon;
+        float f = Math.abs(candidate - compare);
+        boolean ret = Math.abs(candidate - compare) < epsilon;
+        return ret;
```

Comment:
I'm guessing this was put in during debugging? Did you want to undo this one @groombridge34a ?

Reply:
Ooops, my bad. Reverted.